variables.tf

```diff
@@ -39,16 +39,16 @@ variable "additional_ses_sendemail_account_ids" {
   default     = []
 }
 
-variable "aws_region" {
+variable "api_gateway_zone_id" {
   type        = string
-  description = "The AWS region to communicate with."
-  default     = "us-east-1"
+  description = "The Route 53 hosted zone ID for API Gateways."
```

Comment:
Remove unnecessary capitalization:
```suggestion
  description = "The Route 53 hosted zone ID for API gateways."
```

Reply:
Thanks, though in the future, please create separate commits for the capitalization fix and the changes to the alias domain.